openconfig/scripts/generate_wifi_oc_schema.sh

```diff
@@ -25,19 +29,17 @@ export GOPATH=$HOME/go
 YANG_CONVERTER=$GOPATH/src/github.com/openconfig/ygot/generator/generator.go
 
 # OpenConfig modules
```

Comment:
Can you move line 19-22 here so it is closer to the place where it is used?

Reply:
Done.